### ISSUE TITLE
Can't find DataDrivenHelper

### ISSUE THREAD

**Issue by Merurino** [ObjectivityLtd/Test.Automation#88]
Hi, I'm trying to use the DataDrivenHelper class in the latest version for NUnit (3.1.7).

My project can't find the DataDrivenHelper though.

I've even tried the example imports:
https://github.com/ObjectivityLtd/Test.Automation/wiki/NUnit-DataDriven-tests-from-Xml,-CSV-and-Excel-files

using Objectivity.Test.Automation.Tests.NUnit.DataDriven;

It can't find the `Tests` namespace after Automation. Am I missing something?

```
Error	CS0234	The type or namespace name 'Tests' does not exist in the namespace 'Objectivity.Test.Automation' (are you missing an assembly reference?)
````

Regards

**Comment by raczeja** [comment]
Hi,
the DataDrivenHelper class should  be installed with nuget package as content to your project. Please share more details or your solution.

**Comment by Merurino** [comment]
So, it's a simple test project on dotnet framework 4.7.2. I could install this package through nuget but for some reason the DataDrivenHelper would not be accessible. 
I ended up copying the source from the file and it worked for my tests at the time. So don't worry.

I just posted here because maybe somehow the file is not going through somehow, which is unlikely as nuget would package the entire project anyway. Maybe it was something in my project blocking it. So I'll close the issue.

Cheers
